lib.php

```diff
@@ -174,6 +174,16 @@ public function get_links($linkarray) {
         }
         $output = '';
 
+        // Don't show links for certain file types as they won't have been submitted to Turnitin.
```

Comment:
This will only prevent these files from showing Turnitin statuses etc. It won't prevent the files from being processed and sent to Turnitin so I think it may need further work. Although if that latter point is done then this code may not be needed.

I think the right way to do this would be to prevent a submission from being created for feedback_files and introattachment, as from my understanding this is happening and is being entered into the database. Those database rows are then shown on the assignment, and this code will stop that last bit.